test/Main.purs

```diff
@@ -1,22 +1,20 @@
 module Test.Main where
 
 import Prelude hiding (min, max)
-import Control.Monad.Eff (Eff)
-import Control.Monad.Eff.Console (CONSOLE, log)
 import Data.Decimal (Decimal, abs, fromInt, fromNumber, pow, fromString,
                      toNumber, toString, toFixed, acos, acosh, asin, asinh, atan, atanh,
                      atan2, ceil, cos, cosh, exp, floor, ln, log10, max,
                      min, modulo, round, truncated, sin, sinh, sqrt, tan, tanh, e, pi, gamma,
                      toSignificantDigits, isInteger, isFinite, factorial)
-import Data.Maybe (Maybe(..))
-import Test.Assert (ASSERT, assert)
-import Control.Monad.Eff.Random (RANDOM())
-import Control.Monad.Eff.Exception (EXCEPTION())
-import Test.QuickCheck (QC(), quickCheck)
-import Test.QuickCheck.Arbitrary (class Arbitrary, arbitrary)
 import Data.Int as Int
+import Data.Maybe (Maybe(..))
 import Data.Number.Approximate ((≅))
+import Effect (Effect)
+import Effect.Console (log)
 import Math as M
+import Test.Assert (assert)
+import Test.QuickCheck (quickCheck)
+import Test.QuickCheck.Arbitrary (class Arbitrary, arbitrary)
```

Comment:
Sorry for this import noise. My editor integration did this.

I purposefully preserved the formatting of the `Prelude` and `Data.Decimal` imports.